packages/app/components/jsonschemaform/widgets/AnchorWidget.tsx

```diff
@@ -38,7 +38,7 @@ export default function AnchorWidget({ options }: WidgetProps) {
 
     return (
         <a href={href} target="_blank" rel="noopener noreferrer">
-            {options.text || href}
+            <>{(options.text as string) || href}</>
```

Comment:
Was getting a compilation error here as options.text can be a few different types. Assuming this was coming from the new TS version